server-phoenix/priv/repo/migrations/20220608170815_add_author_col_to_events.exs

```diff
@@ -4,6 +4,7 @@ defmodule Helios.Repo.Migrations.AddAuthorColToEvents do
   def change do
     alter table("events") do
       add(:source_author, :string)
+      add(:source_channel, :string)
```

Comment:
We should not be updating existing migrations. If we need to make more changes we need to create a new migration